playground/frontend/playground_components/lib/playground_components.dart

```diff
@@ -40,20 +47,40 @@ export 'src/models/example_view_options.dart';
 export 'src/models/intents.dart';
 export 'src/models/loading_status.dart';
 export 'src/models/outputs.dart';
+export 'src/models/run_shortcut.dart';
 export 'src/models/sdk.dart';
 export 'src/models/shortcut.dart';
 export 'src/models/snippet_file.dart';
 export 'src/models/toast.dart';
 export 'src/models/toast_type.dart';
+
 export 'src/playground_components.dart';
+
 export 'src/repositories/code_client/grpc_code_client.dart';
 export 'src/repositories/code_repository.dart';
 export 'src/repositories/example_client/grpc_example_client.dart';
 export 'src/repositories/example_repository.dart';
+
 export 'src/router/router_delegate.dart';
+
+export 'src/services/analytics/analytics_service.dart';
+export 'src/services/analytics/events/abstract.dart';
+export 'src/services/analytics/events/app_rated.dart';
+export 'src/services/analytics/events/constants.dart';
+export 'src/services/analytics/events/external_url_navigated.dart';
+export 'src/services/analytics/events/feedback_form_sent.dart';
+export 'src/services/analytics/events/report_issue_clicked.dart';
+export 'src/services/analytics/events/run_cancelled.dart';
+export 'src/services/analytics/events/run_finished.dart';
+export 'src/services/analytics/events/run_started.dart';
+export 'src/services/analytics/events/sdk_selected.dart';
+export 'src/services/analytics/events/snippet_reset.dart';
+export 'src/services/analytics/google_analytics4_service/google_analytics4_service.dart';
```

Comment:
We needed an umbrella name for everything that is shared by Playground and ToB. This includes not only services but also widgets and stateful objects. "components" because it is a less specific term than "services", "modules", etc.